public/robots.txt

```diff
@@ -0,0 +1,4 @@
+User-agent: *
+Allow: *
+Disallow: 
+Sitemap: https://cirrus-ci.com/sitemap.xml
```

Comment:
This should allow search engines to crawl all pages making the search ranking higher

Reply:
There is no https://cirrus-ci.com/sitemap.xml and I also think engines are crawling it anyway. `robots.txt` is usually used to block some parts from the engines. As far as I know engines are crawling by default.